components/thumbnail/basic/src/index.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+class ThumbnailBasic extends Component {
+  render () {
+    const { image, captionText } = this.props
+    return (
+      <figure className='sui-ThumbnailBasic'>
+        <img className='sui-ThumbnailBasic-Image' src={image.src} alt={image.alt} />
+        <figcaption className='sui-ThumbnailBasic-Caption'>
+          <span>{captionText}</span>
+        </figcaption>
+      </figure>
+    )
+  }
+}
+
+ThumbnailBasic.displayName = 'ThumbnailBasic'
+
+ThumbnailBasic.propTypes = {
+  image: PropTypes.object.isRequired,
+  captionText: PropTypes.string
```

Comment:
Keep in mind that you have set this to default, but in the render you seem to expect the `captionText` to exist.

Or this is required or you should check if the var exists before creating the figcaption.

Check a similar example:  https://github.com/SUI-Components/sui-components/blob/master/components/card/basic/src/index.js#L38